test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -9,18 +9,18 @@ public class EdgeConfigBuilder
     {
         readonly string deviceId;
         readonly Dictionary<string, IModuleConfigBuilder> moduleBuilders;
-        Option<(string address, string username, string password)> registry;
+        readonly List<(string address, string username, string password)> registries;
 
         public EdgeConfigBuilder(string deviceId)
         {
             this.deviceId = deviceId;
             this.moduleBuilders = new Dictionary<string, IModuleConfigBuilder>();
-            this.registry = Option.None<(string, string, string)>();
+            this.registries = new List<(string, string, string)>();
         }
 
         public void AddRegistryCredentials(string address, string username, string password)
```

Comment:
Who calls this?

Reply:
The individual tests call it when they're building their specific config. I plan to create some higher level function that does this so each test doesn't have to, but for now the tests call it.